### PR TITLE
JOB#1

### PR DESCRIPTION
**Pull request by ArtyomProger**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,28 @@
...
+
+    a+=prompt("Введите аргумент","")
+} 
+var b=a;
+return(b);
+}
+alert(getsum());
+//2-й вариант.Решение в лоб.
+
+function sum(a) {
```
Comment (vitalics) [inline index.js]: If sum would be with 15 arguments, you're will create 15 functions too?

File: index.js [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+  return function(b) {
+     
+      return function(c){
+          return function(d){
+              return a+b+c+d
```
Comment (vitalics) [inline index.js]: Reread learn.js about context and closing. May be you can rewrite toString method(this is native method of object. Functions are objects too)

**Review by vitalics — CHANGES_REQUESTED** [review]
Get sum function is wrong. You need read.more about closing and function context

File: index.js (hunk 1 above)
Comment (ArtyomProger) [reply index.js]: Поэтому я и написал,что такой вариант является неверным.Первый вариант,на мой взгляд,кажется верным.Там только нужно преобразовать конкатенацию в сложение

Comment (vitalics) [inline index.js]: i get you some help. try next code:
``` js
function sum(a){
  // ... implement newFunction here
  newFunction.valueOf = () => a;
  return newFunction;
}
```
test for this function:
``` js
sum(1) == 1 // true 
```

Comment (vitalics) [reply index.js]: why is wrong, I just ask about 15 calls. What would you implement in this case?
sum(1)(2)(3)(4)(5)

**Review by vitalics — APPROVED** [review]

**Pull request merged**